installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -111,8 +122,55 @@ include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+3]
 * xref:../../networking/changing-cluster-network-mtu.adoc#mtu-value-selection_changing-cluster-network-mtu[Changing the MTU for the cluster network]
 * xref:../../networking/changing-cluster-network-mtu.adoc#nw-ovn-ipsec-enable_configuring-ipsec-ovn[Enabling IPsec encryption]
 
+// Opting in to AWS Local Zones
+include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+1]
+
+.Next steps
+
+Choose one of the following options to install an {product-title} cluster in an AWS Local Zone environment:
+
+* xref:../../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Installing a cluster to quickly extend workers to edge compute pools]
+* xref:../../installing/installing_aws/installing-aws-localzone.adoc#creating-aws-local-zone-environment-existing_installing-aws-localzone[Creating an AWS Local Zone environment in an existing VPC]
+
+// Installing a cluster to quickly extend workers to edge compute pools
+include::modules/installation-cluster-quickly-extend-workers.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_aws/installing-aws-localzone.adoc_installing-aws-localzone[Creating the installation configuration file]
+
+* xref:../../installing/installing_aws/installing-aws-localzone.adoc#cluster-limitations-local-zone_installing-aws-localzone[Cluster limitations in AWS Local Zones]
+
+* xref:../../installing/installing_aws/installing-aws-localzone.html#installation-launching-installer_installing-aws-localzone[Deploying your cluster]
```

Comment:
Not sure if you need that reference here. The deploy is a later step, this is presenting and additional before sharing the details of install-config (next section)

Reply:
Agree. Removed.